dataactcore/models/baseInterface.py

```diff
@@ -47,7 +47,7 @@ def __del__(self):
             self.Session.remove()
             self.connection.close()
             self.engine.dispose()
-        except KeyError:
+        except AttributeError:
             # KeyError will occur in Python 3 on engine dispose
             pass
 
```

Comment:
Actually I meant add AttributeError, not replace, I suspect KeyError can still occur.